### PR TITLE
Update NSHD protocol

### PR DESCRIPTION
**Pull request by mpgxvii**
- Add SMWT and GHQ12 to, and remove PHQ8GAD7 from NSHD protocol

### CODE REVIEW

**Review by afolarin — APPROVED** [review]
LGTM

File: NSHD/protocol.json [hunk 1]
```diff
@@ -56,17 +56,68 @@
       }
     },
     {
-      "name": "PHQ8_GAD7",
+      "name": "SMWT",
```
Comment (yatharthranjan) [inline NSHD/protocol.json:59]: Should we name these appropriately since they will be shown in the UI?

File: NSHD/protocol.json [hunk 2]
```diff
@@ -56,17 +56,68 @@
...
+        },
+        "repeatQuestionnaire": {
+          "unit": "min",
+          "unitsFromZero": [660]
+        },
+        "reminders": {
+          "unit": "day",
+          "amount": 0,
+          "repeat": 0
+        }
```
Comment (yatharthranjan) [inline NSHD/protocol.json:106]: Should we only enable notifications for 2 and disable for all others

File: NSHD/protocol.json [hunk 3]
```diff
@@ -56,17 +56,68 @@
...
+        "it": "Richiede uno spazio tranquillo.",
+        "nl": "Vereist een rustige ruimte.",
+        "da": "Kræver et stille rum.",
+        "de": "Benötigt einen ruhigen Platz.",
+        "es": "Requiere un espacio tranquilo."
+      },
+      "estimatedCompletionTime": 1,
+      "protocol": {
+        "repeatProtocol": {
+          "unit": "day",
```
Comment (yatharthranjan) [inline NSHD/protocol.json:95]: month?

**Review by yatharthranjan — COMMENTED** [review]
Hi please find some comments. These apply to all the questionnaires.

File: NSHD/protocol.json (hunk 3 above)
Comment (mpgxvii) [reply NSHD/protocol.json:95]: Should I change it to month now or keep it first for testing?

**Comment by mpgxvii** [comment]
Merging for now for testing, will update the protocol to monthly and fix reminders after testing.

**Pull request merged**